.github/workflows/compat-check-release-release.yml

```
@@ -48,4 +48,3 @@ jobs:
       lightning-version: release
       pennylane-version: release
     secrets: inherit
```

To fix the EOF rendering issue
```suggestion
    secrets: inherit

```